src/doc/rustdoc/src/what-is-rustdoc.md

```diff
@@ -93,6 +93,26 @@ passes `-L`, a flag that helps rustdoc find the dependencies
 your code relies on. If our project used dependencies, we'd get
 documentation for them as well!
 
+## Outer and inner documentation
```

Comment:
adding a space between this heading and the next paragraph would be good, to match the rest of the documents.